app/models/settings.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: settings
+#
+#  id                  :integer          not null, primary key
+#  created_at          :datetime
+#  updated_at          :datetime
+#  singleton_guard     :integer
+#  current_semester_id :integer
+#  npo_app_open        :boolean
+#  student_app_open    :boolean
+#
+
+class Settings < ActiveRecord::Base
+  validates_numericality_of :singleton_guard, equal: 0
+  validates :npo_app_open, inclusion: { in: [true, false] }
+  validates :student_app_open, inclusion: { in: [true, false] }
+  validates :current_semester_id, presence: true
+
+  # source: http://stackoverflow.com/questions/399447/how-to-implement-a-singleton-model
+  def self.instance
+    # there will be only one row, and its ID must be '1'
+    begin
+      first
```

Comment:
Something more complex is basically what we have now, sorry about the confusion. I like your idea of using an initializer; let's do that instead of have a Setting create in seeds.rb.